src/Engine/Physics/Trigger.cpp

```diff
@@ -18,9 +19,10 @@ namespace Physics {
     btScalar Trigger::addSingleResult(btManifoldPoint& cp,
         const btCollisionObjectWrapper* colObj0, int partId0, int index0,
         const btCollisionObjectWrapper* colObj1, int partId1, int index1) {
-        assert(colObj0->getCollisionObject() != trigger);
```

Comment:
Since we are removing this, are we absolutely sure that `colObj1` is the entering body?

Reply:
When I tested, colObj0 was the trigger and colObj1 was the entering body.

Reply:
Hmm, then the parameters may depend on what Bullet does internally. Will have to consult the documentation for details.

Reply:
The documentation doesn't say what the parameters do. Still, it's easy enough for us check which equals the trigger and use the other one for indexing.